api/edge_api/identities/models.py

```diff
@@ -4,18 +4,15 @@
 
 from django.db.models import Prefetch, Q
 from flag_engine.features.models import FeatureStateModel
-from flag_engine.identities.builders import (
-    build_identity_dict,
-    build_identity_model,
-)
-from flag_engine.identities.models import IdentityModel
-from flag_engine.utils.collections import IdentityFeaturesList
+from flag_engine.identities.builders import build_identity_model
+from flag_engine.identities.models import IdentityFeaturesList, IdentityModel
 
 from api_keys.models import MasterAPIKey
 from environments.dynamodb import DynamoIdentityWrapper
 from environments.models import Environment
 from features.models import FeatureState
 from features.multivariate.models import MultivariateFeatureStateValue
+from mappers import map_engine_identity_to_identity_document
```

Comment:
This is true.

Reply:
Are there any options to resolve this? I guess the only option would be to put it back in the engine ... or have another package responsible for it? Maybe defining it twice is the best option.

Reply:
The schemas are defined in engine (with Pydantic models) and now should be the only source of truth about what is stored in the database.

Every `map_engine_{entity}_to_{entity}_document` is a thin wrapper around

1. `BaseModel.dict(entity)`
2. Value type coercion: numeric to `Decimal` and the rest to `str`.

At the moment both Core API and Edge API solve this problem their own way (I personally prefer the value mapper written for Core to marshmallow schemas left in Edge). If we're to factor this logic to a common dependency this will be a very trivial utility. But it surely doesn't belong to the engine, IMO.

Reply:
Ok, I think this is fairly minor for now - let's see where we're at once all this is merged.